src/inmanta/data/dataview.py

```diff
@@ -1268,16 +1269,36 @@ def allowed_filters(self) -> dict[str, type[Filter]]:
         """
         Return the specification of the allowed filters, see FilterValidator
         """
-        return {}
+        return {
+            "managed": BooleanEqualityFilter,
+        }
 
     def get_base_url(self) -> str:
         return "/api/v2/discovered"
 
     def get_base_query(self) -> SimpleQueryBuilder:
         query_builder = SimpleQueryBuilder(
-            select_clause="SELECT environment, discovered_resource_id, values",
-            from_clause=f" FROM {data.DiscoveredResource.table_name()}",
-            filter_statements=["environment = $1"],
+            select_clause="SELECT *",
+            prelude=f"""
```

Comment:
I might be missing something here, but I don't see why you need the prelude.

Reply:
Iirc I was getting `unknown column 'managed'` errors so I mimicked the behaviour from other queries involving filters.